src/composables/routes.js

```diff
@@ -88,6 +88,25 @@ export default memoizeForContainer(({ router, requestData }) => {
     ? publishedFormPath(form.projectId, form.xmlFormId)
     : formPath(form.projectId, form.xmlFormId, 'draft'));
 
+  const newSubmissionPath = (projectIdOrDraft, xmlFormId, draft) => {
```

Comment:
This might be off-topic (I could file a follow-up issue), but could we use `newSubmissionPath()` in `ProjectFormRow` as well? Right now that component is linking to a /- path.

Reply:
changed in `PublicLinkRow` as well